chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationResultNormalizer.java

```diff
@@ -136,6 +137,13 @@ private void setSedTrueIfApplicableToCriteria(CertifiedProductSearchDetails list
                     && certResultRules.hasCertOption(certResult.getCriterion().getId(), CertificationResultRules.SED))
             .forEach(certResult -> certResult.setSed(true));
     }
+    private void setCodeSetsTrueIfApplicableToCriteria(CertifiedProductSearchDetails listing) {
```

Comment:
If an ACB user sets the code set to false during editing will this just set it back to true?

Reply:
Haha!  Noticed this when testing the questionable activity stuff.  Now we only set the value if it null.